x/intertx/go.mod

```diff
@@ -119,7 +119,7 @@ require (
 	golang.org/x/text v0.14.0 // indirect
 	google.golang.org/genproto v0.0.0-20240116215550-a9fa1716bcac // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240125205218-1f4bbc51befe // indirect
-	google.golang.org/protobuf v1.32.0 // indirect
+	google.golang.org/protobuf v1.33.0 // indirect
```

Comment:
This PR updates github.com/golang/protobuf , but it also tries to bump google.golang.org/protobuf to v1.33.0. The latter won't work.